chainstate/storage/src/internal/mod.rs

```diff
@@ -76,6 +76,18 @@ impl<B: storage::Backend> Store<B> {
     pub fn dump_raw(&self) -> crate::Result<storage::raw::StorageContents<Schema>> {
         self.0.dump_raw().map_err(crate::Error::from)
     }
+
+    /// Collect and return all utxos from the storage
+    #[allow(clippy::let_and_return)]
```

Comment:
You can submit an issue about this false positive if you want. 🙃

Reply:
It's already there:
https://github.com/rust-lang/rust-clippy/issues/8114